Earthfile

```diff
@@ -297,7 +297,15 @@ base-image:
         RUN rm /usr/bin/kcrypt
         COPY github.com/kairos-io/kcrypt:$KCRYPT_DEV_BRANCH+build-kcrypt/kcrypt /usr/bin/kcrypt
     END
-
+   # END
+
+   # TEST KCRYPT-CHALLENGER FROM BRANCH
+    ARG KCRYPT_CHALLENGER
```

Comment:
Can't we use only one variable? e.g. use something like `IF [[ ! -z  "$KCRYPT_CHALLENGER_BRANCH" ]]` (not tested).